lib/actions/CodeDeployLambda.js

```diff
@@ -181,15 +181,15 @@ module.exports   = function(SPlugin, serverlessPath) {
               },
               FunctionName: _this.functionName, /* required */
```

Comment:
You could supply this too, and check confirming no other usage, delete where it is set in the constructor.